frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.resources.scenario.ts

```diff
@@ -15,10 +15,7 @@ describe('Test adding/removing discs/nics to/from a VM template', () => {
     .setProvisionSource(provisionSources.Container)
     .build();
 
-  const vm = new VMBuilder(getBasicVMBuilder())
-    .setTemplate(vmt.name)
-    .setName('from-template')
-    .build();
+  const vm = new VMBuilder(getBasicVMBuilder()).setName('from-template').build();
```

Comment:
there is a new flow for creating vm's from templates: through `/k8s/ns/default/virtualization/templates` or `k8s/ns/default/vmtemplates/vm-template-example`. 

We should test the new way of creating vms from templates in all our scenarios

Reply:
we have the test `ID(CNV-4290) Creates VM using VM Template create virtual machine link` in vmtemplate.wizard.scenario.ts

Reply:
yes, but this test tests resources (disks/nics) and should also do that with template flow in mind

Reply:
No need to use template here based on the actual test results.

Reply:
can you please specify, why is there no need to test templates?

Reply:
It does not skip/ignore template test, I mean no need to use `template` field.
```
} else if (template) {
      await this.selectTemplate(template);
    } else {
```

`await this.selectTemplate(template);` will never be executed as we don't have this field. If remove this line and leave this block empty does not make sense and is not accepted by eslint.

To create vm from template should use:
```
      await vmt.createVMFromRowLink();
      await wizard.processWizard(vm.getData());
```
It does not need to have `template` at all.

Reply:
I get that the template option is not in the wizard, but it is still possible to create a VM from the template

Reply:
We have tests to create vm from template and it does not require `template` field to create vm from template.

Reply:
you need to know the name of the template to select it. The implementation (wizard field or clicking on the template row) is less important. 
But having one config in one place is always better IMO.